package.json

```diff
@@ -3,6 +3,7 @@
   "version": "7.0.1-post",
   "description": "A high-level API to control headless Chrome over the DevTools Protocol",
   "main": "./cjs-entry.js",
+  "types": "lib/types.d.ts",
```

Comment:
this `types.d.ts` file isn't published.

https://www.runpkg.com/?puppeteer@7.0.3

![image](https://user-images.githubusercontent.com/1404810/107347581-40e8ce80-6ac6-11eb-80b0-b97df1a806df.png)

Reply:
possibly `"lib/**/*.d.ts"` below doesn't include it since its not nested?

Reply:
Jack fixed this in a subsequent patch release. Does v7.0.3 work for you, @SimenB?

Reply:
Nope, I linked to 7.0.3

Reply:
I see #6844 now, doesn't seem like it's enough

Reply:
Last one, maybe #6848 fixed it?

Reply:
Apologies, I had missed your link earlier. Thanks for bearing with us here @SimenB. @jackfranklin is cutting a release soon.

Reply:
That works, thanks!

New issue, though

```
../../node_modules/puppeteer/lib/types.d.ts:24385:1 - error TS2309: An export assignment cannot be used in a module with other exported elements.

24385 export = puppeteer;
      ~~~~~~~~~~~~~~~~~~~
```

Reply:
As far as I see the `puppeteer` object will be exported as default export *and* as object export in `lib/cjs-entry.js`. I would suggest to define the `puppeteer` object as default export in the types.d.ts only (`export default puppeteer;`). TSC will still accept `import puppeteer from "puppeteer";` then (default import), and in CJS it is still possible to use `const puppeteer = require("puppeteer");` (object import).

Reply:
@SimenB thank you for your help 👍   The namespace idea looks great, I'm looking at creating that PR now.